auth/authorizer.go

```diff
@@ -0,0 +1,81 @@
+package auth
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/casbin/casbin"
+	"github.com/dmarro89/dare-db/logger"
+)
+
+const GUEST_USER = "guest"
+const GUEST_ROLE = "guest"
+const DEFAULT_USER = "admin"
+const DEFAULT_ROLE = "admin"
+
+const RBAC_CONFIG_FILE = "auth/rbac_model.conf"
+const RBAC_POLICY_FILE = "auth/rbac_policy.csv"
```

Comment:
Made path to RBAC files as constants for a better transparency

Reply:
Sorry @vdmitriyev I don't understand, these two paths are already set as constants

Reply:
@vdmitriyev you are right. Sorry for the confusion here. I already addressed it this in this commit to the current PR:  https://github.com/dmarro89/dare-db/commit/2180cf93b0a40a25c2ee516dd84d2c0ca22b7bd7